native/common/jp_context.cpp

```diff
@@ -216,6 +214,7 @@ void JPContext::initializeResources(JNIEnv* env, bool interrupt)
 
 	if (!m_Embedded)
 	{
+        printf("not embedded\n"); fflush(stdout);
```

Comment:
:eyes:

Reply:
This PR is still a work in progress.  So still a lot of rough edges.  I have yet to come up with a logical reason that jpype needs to be started on main thread rather than on the spawned python thread.  And i need to resolve how to prevent the user from calling shutdown.   

That is unfortunately one of the key architectural problems with jpype.  Shutdown should never have been exposed in the first place.  The entry point should have spawned a thread, the original thread should have attached, then the launch thread should call destroy jvm.  Modules in Python don't generally unload themselves or have time bombs to render themselves unusable, but that is what shutdown does.

  Unfortunately like the calling classpath from startJVM as "-Dclasspath" which should never be done(it prevents our jar from being loaded requiring sideloading with a URLClassLoader), the shutdownJVM is in all the old docs and thus ilI will forever be forced to deal with.